test/system/420-cgroups.bats

```diff
@@ -52,6 +52,17 @@ load helpers
     # --cgroupns=host is required to have full visibility of the cgroup path inside the container
     run_podman run --cgroups=disabled --cgroupns=host --rm $IMAGE cat /proc/self/cgroup
     is "$output" $current_cgroup "--cgroups=disabled must not change the current cgroup"
+
+    ctr1="c1-$(safename)"
+    ctr2="c2-$(safename)"
+
+    # verify that "podman stats --all" works when there is a container with --cgroups=disabled
+    run_podman run --cgroups=disabled --name $ctr1 -d $IMAGE top
+    run_podman run --name $ctr2 -d $IMAGE top
+
+    run_podman stats -a --no-stream --no-reset
```

Comment:
maybe as extra check to not just rely on the exit code

```
assert "$output" !~ "$ctr1" "ctr1 not in stats output"
assert "$output" =~ "$ctr2" "ctr2 in stats output"
```

Reply:
thanks.

Added and pushed a new version